examples/workflows/plot_simulate_evoked.py

```diff
@@ -60,35 +61,35 @@
 
 ###############################################################################
 # First, we add a distal evoked drive
-weights_ampa_d1 = {'L2_basket': 0.006562, 'L2_pyramidal': .000007,
-                   'L5_pyramidal': 0.142300}
-weights_nmda_d1 = {'L2_basket': 0.019482, 'L2_pyramidal': 0.004317,
-                   'L5_pyramidal': 0.080074}
+weights_ampa_d1 = {'L2_basket': 0.061, 'L2_pyramidal': 1.155,
+                   'L5_pyramidal': 1.004}
+weights_nmda_d1 = {'L2_basket': 0.021, 'L2_pyramidal': 0.299,
+                   'L5_pyramidal': 0.579}
```

Comment:
I know Samika already spent some time in tuning and finding these parameters, but random thought: how about saving the old dipole simulations using `dpl.save` and then using the optimization function to find these parameters:

https://github.com/jonescompneurolab/hnn-core/pull/77/files

that way you can review that PR as well while working on this

Reply:
Worth a shot, although I may run SNPE on this as well to see what gets the best fit.

Reply:
Holy ~beep~ some of these weights have increased by 5 orders of magnitude!! Has anyone checked that they still make any sense physiologically?

Reply:
Agreed, the discussion here may help the interpretation: https://www.neuron.yale.edu/phpBB/viewtopic.php?t=1187

I am going to run some optimization and see the range of values that satisfy the dipole waveforms. I am fairly confident there are diminishing returns on increasing conductance which may explain these huge values. Unlike real cells, our neurons can't explode from osmotic pressure when a bunch of ions come flooding in.

Reply:
> Unlike real cells, our neurons can't explode from osmotic pressure when a bunch of ions come flooding in.

😂😂 Yes, most likely diminishing returns. I'm very curious to learn what you learn, feel free to share a gist 😉